specs/electra/p2p-interface.md

```diff
@@ -32,28 +29,6 @@ Topics follow the same specification as in prior upgrades.
 
 The `beacon_block` topic is modified to also support Electra blocks.
 
-The `beacon_aggregate_and_proof` and `beacon_attestation_{subnet_id}` topics are modified to support the gossip of the new attestation type.
-
 The specification around the creation, validation, and dissemination of messages has not changed from the Capella document unless explicitly noted here.
 
 The derivation of the `message-id` remains stable.
-
-#### Global topics
-
-##### `beacon_aggregate_and_proof`
-
-The following convenience variables are re-defined
-- `index = get_committee_indices(aggregate.committee_bits)[0]`
-
-The following validations are added:
-* [REJECT] `len(committee_indices) == 1`, where `committee_indices = get_committee_indices(aggregate)`.
-* [REJECT] `aggregate.data.index == 0`
-
-##### `beacon_attestation_{subnet_id}`
-
-The following convenience variables are re-defined
-- `index = get_committee_indices(attestation.committee_bits)[0]`
-
-The following validations are added:
-* [REJECT] `len(committee_indices) == 1`, where `committee_indices = get_committee_indices(attestation)`.
-* [REJECT] `attestation.data.index == 0`
```

Comment:
the key thing of EIP-7549 is to have `data.index = 0` so we should not remove this? same to above